src/github.com/matrix-org/go-neb/services/rssbot/rssbot.go

```diff
@@ -34,6 +35,16 @@ var (
 
 const minPollingIntervalSeconds = 60 * 5 // 5 min (News feeds can be genuinely spammy)
 
+// includeRules contains the rules for including or excluding a feed item.
+type includeRules struct {
```

Comment:
You also need to state how include/exclude clashes are resolved. If I include author "foo" and exclude title "bar" and an article has both author Foo and title Bar, does it get included or excluded?

Reply:
Added to the docs.